roles/approvals/meta/main.yml

```diff
@@ -7,14 +7,14 @@ galaxy_info:
 
   license: MIT
 
-  min_ansible_version: 2.2
+  min_ansible_version: '2.2'
 
   platforms:
     - name: Ubuntu
       versions:
-        - 18.04
+        - 'bionic'
+        - 'jammy'
```

Comment:
why do we keep version jammy?

Reply:
Updated to only include Jammy, and not Bionic, since Jammy is the version of Ubuntu we know these roles run on successfully.